packages/dumb/src/components/settings/history/history-table-content.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2023, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { HistorySpannedRow } from './history-table-spanned-row'
+import { HistoryTableRow } from './history-table-row'
+import React, { FunctionComponent } from 'react'
+import { TimePrefs } from 'medical-domain'
+import { IncomingRow } from '../../../models/historized-parameter.model'
+import { buildAllRows } from './history-table.util'
+
+interface HistoryTableContentProps {
+  onSwitchToDaily: Function
+  rows: IncomingRow[]
+  timePrefs: TimePrefs
+  length: number
+}
+
+export const HistoryTableContent: FunctionComponent<HistoryTableContentProps> = (props): JSX.Element => {
+  const { onSwitchToDaily, rows, timePrefs, length } = props
+  const rs = buildAllRows(rows, timePrefs)
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏It would be nicer to have a more readable variable name, `rs` is not very understandable

```suggestion
  const rows = buildAllRows(rows, timePrefs)
```

Reply:
cannot set rows as it already used however i will put something else